src/routes/download/all-scheme-payment-data.js

```diff
@@ -0,0 +1,19 @@
+import Wreck from '@hapi/wreck'
+import { buildBackendUrl } from '../../api/build-backend-url.js'
+
+export const downloadAllSchemePaymentData = {
+  method: 'GET',
+  path: '/all-scheme-payment-data/file',
+  handler: async function (_request, h) {
+    const backendUrl = buildBackendUrl('/file')
+    const stream = await Wreck.request('get', backendUrl)
```

Comment:
I would still have this abstracted into the API directory.  Perhaps in a getStream module to make it's use clearer and distinct to the standard get.